common/app/common/commercial/hosted/HostedGalleryPage.scala

```diff
@@ -22,6 +22,7 @@ case class HostedGalleryPage(
 
   val pageTitle: String = s"Advertiser content hosted by the Guardian: $title - gallery"
   val twitterText = shareText.getOrElse(if(standfirst.length < 136) standfirst else title) + " #ad"
+  val imageUrl = images.head.url
```

Comment:
Ha, it is definitely a minimalist one! I just recall a bug where we were getting `.tail` on an empty list and it broke the page so always worth checking :)